config/default..sample-with-domain.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Looks like two consecutive dots in the filename.  Intended?

Reply:
Nope, fixed it.